COMPILING.md

```diff
@@ -114,6 +114,25 @@ files.
    complete set of built binaries can be found in `./build/bin` once the build
    is complete.
 
+## Compilation on Windows using Visual Studio
+
+Follow these instructions to work on Windows with Visual Studio:
+
+1. First, clone the `cbmc` repository locally.
+2. Open the `cbmc` folder into Visual Studio. Visual Studio 2017 and
+   later have automated support for CMake projects, so you need to
+   give sometime to Visual Studio to index the project and load its
+   own plugins, and then it's going to be ready to build `cbmc`.
+3. Once indexing and plugin loading has finished, a menu `Build`
+   should have appeared on the top bar. From there, select `Build All`.
+4. After the build has finished, there should be a folder `out` present.
+   Navigating `out/build/<build_parameters>/bin` should get you to the
+   binaries and other artifacts built, with `<build_parameters>` corresponding
+   to something like `x64-Debug (Default)` or whatever the equivalent is
+   for your system.
+
+The above instructions hae been tested against Visual Studio 2019.
```

Comment:
Above says VS 2017, which version is it?

Reply:
I have tested it with Visual Studio 2019, but 2017 is the first version that added support for this.

Do I convey this accurately through the text?

Reply:
I am OK with the two different version numbers above, as the versions match the facts. Maybe add 2019 to the section title though.